metricbeat/helper/interfacer.go

```diff
@@ -1,3 +1,17 @@
+/**
+
+
+= MetricSeter
+
+== Timeout
+Each metricseter must implement on his side to make sure the timeout is followed.
+Otherwise it can lead to the issue that multiple calls for one metricset happen in parallel and pile up.
+
+The initial idea was to use a Cancel function, but it was not implemented as it would add complexity by requiring
+an identifier for each fetch call and it would have to be decide, which fetch call must be canceled.
```

Comment:
I was also not sure where to put it. I thin it is important to have it document somewhere to point to, as the discussion will pop up again in the future. User docs is not the right place. It would be good to have it in the source code so the in case someone wants to change it, he sees this comment. I had it first with the Timeout config option, but it was too long there. Some other ideas?